api/sip/__init__.py

```diff
@@ -13,7 +15,7 @@ class SIP2AuthenticationProvider(BasicAuthenticationProvider):
     DATE_FORMATS = ["%Y%m%d", "%Y%m%d%Z%H%M%S", "%Y%m%d    %H%M%S"]
 
     def __init__(self, server, port, login_user_id,
-                 login_password, location_code, field_separator,
+                 login_password, location_code, field_separator='|',
```

Comment:
It's almost always '|' and libraries generally don't even know it's configurable, so we shouldn't make libraries specify this parameter.